server/src/Controller/Channel/ChannelController.js

```diff
@@ -0,0 +1,99 @@
+import {
+  create,
+  deleteDoc,
+  findOne,
+  findMany,
+  update,
+} from '../../Repositories/genericRepository';
+import successHandler from '../../libs/routes/successHandler';
+import { channelModel } from '../../Model';
+
+
+const getChannel = async (req, res, next) => {
+  try {
+    const { id } = req.params;
+    const result = await findOne(channelModel, id);
+    if (!result) {
+      const err = new Error('Channel does not exists');
```

Comment:
Ideally, we should return 404 if the channel is not found. This text `Channel does not exist` can be moved to constants

Reply:
Okay, I will change it.